src/compas_model/elements/element.py

```diff
@@ -5,723 +5,402 @@
-        """Display schema of the element used for the viewer.
+    def _copy_geometries(self, geometries, copy_flag):
+        """
+        Helper function to copy geometries.
 
         Returns
         -------
-        dict
-            The display schema of the element.
-
+        list
         """
-        ordered_dict = OrderedDict(
-            [
-                (
-                    "geometry_simplified",
-                    {
-                        "facecolor": [0.0, 0.0, 0.0],
-                        "linecolor": [0.0, 0.0, 0.0],
-                        "linewidth": 3,
-                        "opacity": 1.0,
-                        "edges": True,
-                        "is_visible": True,
-                    },
-                ),
-                (
-                    "geometry",
-                    {
-                        "facecolor": [0.9, 0.9, 0.9],
-                        "linecolor": [0.0, 0.0, 0.0],
-                        "linewidth": 1,
-                        "opacity": 0.75,
-                        "edges": True,
-                        "is_visible": True,
-                    },
-                ),
-                (
-                    "frame",
-                    {
-                        "facecolor": [0.0, 0.0, 0.0],
-                        "linecolor": [1.0, 1.0, 1.0],
-                        "linewidth": 1,
-                        "opacity": 1.0,
-                        "edges": True,
-                        "is_visible": False,
-                    },
-                ),
-                (
-                    "aabb_mesh",
-                    {
-                        "facecolor": [0.0, 0.0, 0.0],
-                        "linecolor": [1.0, 1.0, 1.0],
-                        "linewidth": 1,
-                        "opacity": 0.25,
-                        "edges": True,
-                        "is_visible": False,
-                    },
-                ),
-                (
-                    "oobb_mesh",
-                    {
-                        "facecolor": [0.0, 0.0, 0.0],
-                        "linecolor": [1.0, 1.0, 1.0],
-                        "linewidth": 1,
-                        "opacity": 0.25,
-                        "edges": True,
-                        "is_visible": False,
-                    },
-                ),
-                (
-                    "convex_hull",
-                    {
-                        "facecolor": [0.0, 0.0, 0.0],
-                        "linecolor": [1.0, 1.0, 1.0],
-                        "linewidth": 1,
-                        "opacity": 0.25,
-                        "edges": True,
-                        "is_visible": False,
-                    },
-                ),
-            ]
+        return (
+            [g.copy() if g and copy_flag else g for g in geometries]
+            if isinstance(geometries, list)
+            else [geometries.copy()]
+            if copy_flag
+            else [geometries]
```

Comment:
write this out properly. difficult for other devs to work with...

Reply:
Replaced with a proper style.